### PR TITLE
Ec2 no append dimension test

### PR DESCRIPTION
**Pull request by ChenaLee**
# Description of the issue
Ec2 no append dimension needs to emit 'host' dimension

# Description of changes
added test for no append dimensions

NOTE: this change was added on top of changes in https://github.com/aws/amazon-cloudwatch-agent-test/pull/26. A lot of commits are showing here because of that. Ignore them, and I will update PR after that one merges. Changes for this PR starts from https://github.com/aws/amazon-cloudwatch-agent-test/pull/41/commits/0f6afb09cc512217db2826ec61ed0fd10f874174

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
Github actions e2e test

### CODE REVIEW

File: environment/MetaData.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (SaxyPandaBear) [inline environment/MetaData.go:1]: nit - the file name shouldn't be capitalized

**Review by SaxyPandaBear — COMMENTED** [review]
@ChenaLee should I hold off on reviewing this until the other PR merges?

**Comment by SaxyPandaBear** [comment]
This has a lot of conflicts from the other PRs that flew in

**Comment by williazz** [comment]
Could you please share the link to the GitHub action run on your fork?

**Comment by SaxyPandaBear** [comment]
merge conflicts need to be addressed

**Pull request closed without merging**